megamek/src/megamek/common/Terrain.java

```diff
@@ -494,12 +526,24 @@ public int movementCost(Entity e) {
                 } else {
                     mp = 1;
                 }
+
+                if (isCrossCountry) {
+                    if (((moveMode == EntityMovementMode.HOVER) || (moveMode == EntityMovementMode.TRACKED))
+                            && (level == 2)) {
+                        mp *= 2;
+                    } else if ((moveMode == EntityMovementMode.WHEELED)) {
+                        mp *= 2;
+                    }
```

Comment:
Is this meant to be the same? If so, maybe look at using an or statement?

Reply:
Wheeled vehicles aren't allowed in rough while all ground vehicles are not allowed in ultra rough.